src/main/java/org/dataconservancy/packaging/tool/cli/RdfContentProvider.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2016 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataconservancy.packaging.tool.cli;
+
+import org.apache.jena.rdf.model.*;
+
+import org.dataconservancy.packaging.shared.ContentProvider;
+import org.dataconservancy.packaging.tool.model.ipm.FileInfo;
+import org.dataconservancy.packaging.tool.model.ipm.Node;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * Implementation of ContentProvider for RDF sourced data.
+ * Created by Ben Trumbore on 12/1/2016.
+ */
+public class RdfContentProvider extends ContentProvider {
+
+    private Logger  LOG = LoggerFactory.getLogger(RdfContentProvider.class);
+    private Model   domainObjects = null;
+    private URI     contentPath;
+
+    /**
+     * Create a content provider that can be passed to IpmPackager.buildPackage.
```

Comment:
This is also new to me (I have more experience with doxygen than with javadoc).  I added the link.